### PR TITLE
Graph total endpoints

### PR DESCRIPTION
**Pull request by YarikRevich**
Closes these:
- Closes https://github.com/0chain/system_test/issues/420

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | feature/atlus_total_endpoints  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | feature/snapshots  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | feature/atlus_total_endpoints  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | feature/snapshot-batch-update  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

File: tests/api_tests/atlus_chimney_test.go [hunk 1]
```diff
@@ -0,0 +1,2014 @@
...
+				client.HttpOkStatus)
+			require.Nil(t, err)
+			require.NotNil(t, resp)
+			require.NotZero(t, *getGraphTotalLocked)
+
+			return !cmp.Equal(*getGraphTotalLocked, getGraphTotalLockedBefore)
+		})
+	})
+
+	t.RunWithTimeout("Check if a graph of total locked increases after write pool creation, should work", time.Minute*6, func(w *test.SystemTest) {
```
Comment (devyetii) [inline tests/api_tests/atlus_chimney_test.go:276]: Duplicate of the one before

**Review by stewartie4 — COMMENTED** [review]
@YarikRevich what is the status of this PR
What's blocking it?

**Comment by Kishan-Dhakan** [comment]
@stewartie4, now @devyetii is working on this PR.

**Review by stewartie4 — CHANGES_REQUESTED** [review]
closing in place of https://github.com/0chain/system_test/pull/593

**Pull request closed without merging**